engine/compiler/util.go

```diff
@@ -190,6 +188,48 @@ func isRestrictedVariable(env map[string]*manifest.Variable) bool {
 	return false
 }
 
+// Upper value for cpu and memory requests. It is same as stage level
+// cpu & memory requests.
+func getStepUpperRequestVal(r resource.StageResources) ResourceObject {
+	return ResourceObject{
```

Comment:
@bradrydzewski Shall we make stage level request mandatory? Or shall we set some default value for stage level memory/cpu?

Reply:
we can re-purpose these request variables:
https://github.com/drone-runners/drone-runner-kube/blob/master/engine/compiler/compiler.go#L53

these variables are currently used to define per-step defaults.  We can ignore per-step resource requests, and instead use stage-level resource requests.

Reply:
Fixed